src/vcpkg-test/arguments.cpp

```diff
@@ -27,8 +26,7 @@ TEST_CASE ("VcpkgCmdArguments from lowercase argument sequence", "[arguments]")
 
     REQUIRE(*v.vcpkg_root_dir == "C:\\vcpkg");
     REQUIRE(*v.scripts_root_dir == "C:\\scripts");
-    REQUIRE(*v.builtin_ports_root_dir == "C:\\ports");
-    REQUIRE(*v.builtin_registry_versions_dir == "C:\\versions");
+    REQUIRE(*v.registry_root_dir == "C:\\registry");
```

Comment:
There should be a test for "what is `*v.registry_root_dir` when `--x-registry-root` isn't passed"